mp/src/game/shared/momentum/mom_gamerules.cpp

```diff
@@ -347,17 +347,15 @@ void CMomentumGameRules::RadiusDamage(const CTakeDamageInfo &info, const Vector
     const int MASK_RADIUS_DAMAGE = MASK_SHOT & (~CONTENTS_HITBOX);
     CBaseEntity *pEntity = nullptr;
     trace_t tr;
-    float falloff;
     Vector vecSpot;
-
     Vector vecSrc = vecSrcIn;
 
-    if (flRadius)
-        falloff = info.GetDamage() / flRadius;
-    else
-        falloff = 1.0f;
-
-    CBaseEntity *pInflictor = info.GetInflictor();
+    // NOTE: this is the falloff value used for rocket self-damage in TF2,
+    // for enemies the falloff is calculated using (damage / radius).
+    // Since Momentum has no enemies this should be fine,
+    // unless there's something like a map entity which requires
+    // a specific amount of damage.
+    float falloff = 0.5f;
```

Comment:
Regarding your comment then, would it be beneficial to put this behind a `GameModeIs(GAMEMODE_RJ)` check?

Reply:
Added gamemode check, using `damage / radius` for falloff by default.